queries/t32/highlights.scm

```diff
@@ -0,0 +1,134 @@
+[
+  "="
+  "^^"
+  "||"
+  "&&"
+  "+"
+  "-"
+  "*"
+  "/"
+  "%"
+  "|"
+  "^"
+  "=="
+  "!="
+  ">"
+  ">="
+  "<="
+  "<"
+  "<<"
+  ">>"
+  ".."
+  "--"
+  "++"
+  "+"
+  "-"
+  "~"
+  "!"
+  "&"
+  "->"
+  "*"
+] @operator
+
+[
+ "("
+ ")"
+ "{"
+ "}"
+ "["
+ "]"
+] @punctuation.bracket
+
+[
+  ","
+  "."
+  ";"
+] @punctuation.delimiter
+
+[
+  (access_class)
+  (address)
+  (bitmask)
+  (file_handle)
+  (frequency)
+  (integer)
+  (percentage)
+  (time)
+] @number
+
+(float) @float
+
+(string) @string
+
+(path) @string.special
+
+(symbol) @string.special
```

Comment:
Perhaps`@symbol` applies here. But I don't know the scripting language

Reply:
Thank you for your feedback. `(symbol)` in this context refers to a debug symbol that is part of a binary for an embedded target. Most of the time debug symbols are acting as a shorthand name for a specific memory address or address range, so symbol probably makes sense here.

On the other hand, paths are very similar to a string without any delimiters. The PRACTICE parse uses the command context to determine whether the next argument is supposed to be a file path and accepts both a string and a path.